### ISSUE TITLE
RPi Pico W adapter to UNO: I hope to mount the Pico W to the back side

### ISSUE THREAD

**Issue by kenneth558** [solderparty/flexypin_adapters_hw#5]
Can the FlexyPins be installed upside-down, so to speak, on the back side of the adapter board to allow the RPi Pico W to be located farther from and more shielded to/from my board?  I realize then that the bootsel button is not accessible, but are the adapter board's FlexyPin pads and solder mask capable of backside mounting?

**Comment by arturo182** [comment]
Yes, you should be able to solder the FlexyPins facing downward, the footprint for that is the same on both sides.

However, I'm not sure if they will be able to hold the Pico in place since it won't be flush with the adapter anymore (the top of the Pico will be facing the bottom of the Adapter board). 

The USB port and the BOOTSEL button on the Pico will align with the cutout in the Adapter, but the RP2040 and the crystal will add a mm or so.

I've never tried it, so it's hard to tell without testing 😅 I'd say, most likely it will work. 

Note that if you are using the Pico W, the WiFi performance might be decreased because of the Adapter now being in the way.

**Comment by kenneth558** [comment]
" the Adapter now being in the way." ?
I thought it would be the other way around, but I just want to put the Pico W on the outside of the entirety, whether it means top side or bottom.  Thank you!!!

**Comment by Anathae** [comment]
It might be better to design a separate carrier board entirely for the RP Pico W, as the debug pins have moved inboard, and for best wireless connectivity a keep out zone is needed at that antenna end of the Pico W.  In the current documentation on hardware development for the RP2040, the RPi foundation includes a VGA board for both of the RP Pico boards.  For the Pico W, they have a 9mm by 14mm keep out zone under the antenna area of the Pico W.

**Comment by kenneth558** [comment]
What I've done so far is to put headers on the Pico W which allows me to solder the headers either to extend upward or extend downward at the expense of project height.  Of course then pins go on this solderparty adapter.
